plugins/scaffolder-backend/src/scaffolder/jobs/processor.ts

```diff
@@ -120,7 +120,7 @@ export class JobProcessor implements Processor {
           // Log to the current stage the error that occurred and fail the stage.
           stage.status = 'FAILED';
           logger.error(`Stage failed with error: ${error.message}`);
-
+          logger.debug(error.stack);
```

Comment:
I'm a bit torn on this one. If it's at debug level, we just have to make sure to return the corresponding information (or at least rich enough information) in the logs instead, so that it's clearly visible where and how the error happened.

Reply:
We'll address this in a future PR, but for now people would have to run in debug mode to troubleshoot an issue.